### DISCUSSION TITLE
泪目了朋友们

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#5408] [其他]
<img src="https://img-static.mihoyo.com/communityweb/upload/2bc65c226456bea0398f59387142d6cf.png" alt="帕姆 震惊" width="75" >
万恶的学校星期六不休息，我们要连上8天课

**Comment by 1luik** [comment]
6

**Comment by KELERSAMA** [comment]
同乐

**Comment by cpu644** [comment]
？不是哥们

**Comment by summer293** [comment]
啊？

**Comment by BJ-cmyk** [comment]
哈哈～我就不一样了，我的学校只要求我们上连续两周的课😅👍
![1727165032985](https://github.com/user-attachments/assets/2d1fdef5-6c9d-4f52-a49c-1ad9cac37969)
